### ISSUE TITLE
AssertionError: 2 != 0 : Not all expected timeslots deleted

### ISSUE THREAD

**Issue by larseggert** [ietf-tools/datatracker#4806]
### Describe the issue

A lot of CI runs lately are failing like this:
```
======================================================================
FAIL: test_delete_time_interval (ietf.meeting.tests_js.EditTimeslotsTests)
Delete button for a time interval should delete all timeslots in that interval
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/datatracker/datatracker/ietf/meeting/tests_js.py", line 1667, in test_delete_time_interval
    self.do_delete_time_interval_test(cancel=False)
  File "/__w/datatracker/datatracker/ietf/meeting/tests_js.py", line 1663, in do_delete_time_interval_test
    self.do_delete_test(selector, keep, delete, cancel)
  File "/__w/datatracker/datatracker/ietf/meeting/tests_js.py", line 1594, in do_delete_test
    self.assertEqual(
AssertionError: 2 != 0 : Not all expected timeslots deleted
```
See https://github.com/ietf-tools/datatracker/actions/runs/3581847477/jobs/6025382408

Do we know why and can this be fixed?

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by jennifer-richards** [comment]
Not sure why, but there's some time zone gymnastics involved in the test. I will poke at it.

**Comment by jennifer-richards** [comment]
There's something strange going on. I ran the test many times. It passed a dozen or so times, then started failing consistently. I added a `debug.show()` to output the meeting time zone and over a couple dozen consecutive runs, the zone was `Pacific/Majuro`. The test failed every time. When I added some calls to `Faker().timezone()` before creating the test meeting, the time zone became random again and the tests started passing. With the `Faker().timezone()` calls removed again the test continues to pass every time.

If I explicitly set `self.meeting.time_zone = 'Pacific/Majuro'`, the test fails.

That time zone is at GMT + 12 which makes date calculations likely to be at fault for the failure. However, I'm more concerned with the bizarre, inconsistent behavior of the faker.
